latest/index.bs

```diff
@@ -175,7 +175,294 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data. OME-NGFF tables are compatible with the [AnnData model](https://github.com/theislab/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
+│
+└── 123.zarr
+    └── table                 # The table group is a container which holds a table that is compatible with AnnData.
+        │                     # The table group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image, labels, or points group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "@type", which is set to `"ngff:region_table"`
+        |                     # `.zattrs` MUST contain "region", which is the path to the data the table is annotating.
+        |                     # "region" MUST be a single path (single region) or an array of paths (multiple regions).
+        |                     # "region" paths MUST be objects with a key "path" and the path value MUST be a string.
+        |                     # `.zattrs` MUST contain "region_key" if "region" is an array. "region_key" is the key in `obs` denoting which region a given row corresponds to.
```

Comment:
This review comment was from months ago (forgot to send), I prefer the full path approach since it is less ambiguous, and the workaround for rows annotating multiple regions can still be achieved with full paths, as I described in this comment: https://github.com/ome/ngff/pull/64#issuecomment-1439194263

Reply:
I think we have used relative paths everywhere else in the NGFF spec to date. This is preferable in the case that you generate the data in one location, then upload it to a different location/server which you don't know about when you generate the data.